Automattic/jetpack
# Boost: fix inconsistent lazy load

Fixes #21585

#### Changes proposed in this Pull Request:
* The initial configuration of boost lazy-load accurately get the status from jetpack plugin

#### Jetpack product discussion
N/A

#### Does this pull request change what data or activity we track or use?
No

#### Testing instructions:
* Get a fresh site with jetpack installed
* Make sure lazy loading is disabled in **Jetpack > Settings > Performance**
* Install boost and go to boost settings page
* Refresh the page
* Make sure lazy-loading stays disabled
